backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExport.kt

```diff
@@ -182,31 +182,31 @@ internal class ObjCExport(val context: Context, symbolTable: SymbolTable) {
 
         """.trimIndent())
 
-        val uiDeviceFamilyValues = if (target.isTvOs) {
-            """
-                |<integer>3</integer>
-            """.trimIndent()
+        // 1 - iPhone
+        // 2 - iPad
+        // 3 - AppleTV
+        val uiDeviceFamilyValues = if (target.isTvOsBased) {
```

Comment:
This will anyway turn into `when` over target (or target family) after adding watchOS, making `isTvOsBased` unnecessary. Then why not just use this fact?